src/main/java/com/bootcamp/demo/model/component/ScooterStatus.java

```diff
@@ -10,6 +10,16 @@
         this.value = value;
     }
 
+    public static boolean contains(String test) {
```

Comment:
Is this method used? If you need such a method it can be rewritten in a simpler way:
`        return asList(ScooterStatus.values()).contains(test);`
where you can extract the list creation as a class constant
or:
       ` return EnumSet.of(AVAILABLE, RESERVED).contains(test)`